knowledge-base/progressbar-add-multiple-labels.md

```diff
@@ -0,0 +1,76 @@
+---
+title: Add Multiple Labels in a Blazor ProgressBar
+description: Learn how to display two or more labels in a ProgressBar for Blazor.
+type: how-to
+page_title: How to Add Multiple Labels in a Blazor ProgressBar
+slug: progressbar-kb-add-multiple-labels
+tags: progressbar, blazor, two, multiple, label, template, css
+res_type: kb
+ticketid: 1659413
+---
+
+## Environment
+
+<table>
+    <tbody>
+        <tr>
+            <td>Product</td>
+            <td>ProgressBar for Blazor</td>
+        </tr>
+    </tbody>
+</table>
+
+## Description
+
+I want to display two labels on my ProgressBar component: one on the left side to show the current progress and another on the right side for the remaining value. 
+
+
+This KB article also answers the following questions:
+- How can I customize the label inside a ProgressBar in Blazor?
+- Is it possible to display two or more labels in a ProgressBar?
+- How do I use the label template feature in the ProgressBar for Blazor?
+
+## Solution
+
+To display two or more labels within a [ProgressBar]({%slug progressbar-overview%}) for Blazor, utilize the [Label Template]({%slug progressbar-label%}#template):
+* Declare the `Template` inside the `ProgressBarLabel` label tag.
+* Add your desired labels in separate HTML containers.
+* Use CSS to position them based on your preferences.
+
+Here is an example of how to implement two labels inside a ProgressBar:
+
+````CSHTML
+<style>
```

Comment:
Consider adding the CSS code after the component, so that the reader can make the connection between the component `Class` and the CSS code more easily.